msi/GitProduct.wxs

```diff
@@ -27,13 +28,64 @@
     <!-- Other MSI specific properties -->
     <Property Id='MSIFASTINSTALL' Value='7' />
 
+    <!-- Git for Windows MSI Properties -->
+    <!-- CLIENVIRONMENT Property implementing the "Remember Property" pattern -->
+    <Property Id="CLIENVIRONMENT" Value="Bash">
+      <RegistrySearch Id="CliEnvironmentSearch" Root="HKLM" Key="SOFTWARE\GitForWindows" Name="CliEnvironment" Type="raw" />
+    </Property>
+    <SetProperty Id="CMDLINE_CLIENVIRONMENT" Value="[CLIENVIRONMENT]" Before="AppSearch" Sequence="first" />
+    <SetProperty Id="CLIENVIRONMENT" Value="[CMDLINE_CLIENVIRONMENT]" After="AppSearch" Sequence="first">CMDLINE_CLIENVIRONMENT</SetProperty>
+
+    <!-- LINEENDINGS Property implementing the "Remember Property" pattern -->
+    <Property Id="LINEENDINGS" Value="ConvertOnCheckout">
+      <RegistrySearch Id="LineEndingsSearch" Root="HKLM" Key="SOFTWARE\GitForWindows" Name="LineEndings" Type="raw" />
+    </Property>
+    <SetProperty Id="CMDLINE_LINEENDINGS" Value="[LINEENDINGS]" Before="AppSearch" Sequence="first" />
+    <SetProperty Id="LINEENDINGS" Value="[CMDLINE_LINEENDINGS]" After="AppSearch" Sequence="first">CMDLINE_LINEENDINGS</SetProperty>
+
+    <!-- TERIMNAL Property implementing the "Remember Property" pattern -->
+    <Property Id="TERMINAL" Value="MinTTY">
+      <RegistrySearch Id="TerminalSearch" Root="HKLM" Key="SOFTWARE\GitForWindows" Name="Terminal" Type="raw" />
+    </Property>
+    <SetProperty Id="CMDLINE_TERMINAL" Value="[TERMINAL]" Before="AppSearch" Sequence="first" />
+    <SetProperty Id="TERMINAL" Value="[CMDLINE_TERMINAL]" After="AppSearch" Sequence="first">CMDLINE_TERMINAL</SetProperty>
+
+    <!-- ENABLEFSCACHE Property implementing the "Remember Property" pattern -->
+    <Property Id="ENABLEFSCACHE" Value="1">
+      <RegistrySearch Id="EnableFsCacheSearch" Root="HKLM" Key="SOFTWARE\GitForWindows" Name="EnableFsCache" Type="raw" />
+    </Property>
+    <SetProperty Id="CMDLINE_ENABLEFSCACHE" Value="[ENABLEFSCACHE]" Before="AppSearch" Sequence="first" />
+    <SetProperty Id="ENABLEFSCACHE" Value="[CMDLINE_ENABLEFSCACHE]" After="AppSearch" Sequence="first">CMDLINE_ENABLEFSCACHE</SetProperty>
+
+    <!-- ENABLEGCM Property implementing the "Remember Property" pattern -->
+    <Property Id="ENABLEGCM" Value="1">
+      <RegistrySearch Id="EnableGcmSearch" Root="HKLM" Key="SOFTWARE\GitForWindows" Name="EnableGcm" Type="raw" />
+    </Property>
+    <SetProperty Id="CMDLINE_ENABLEGCM" Value="[ENABLEGCM]" Before="AppSearch" Sequence="first" />
+    <SetProperty Id="ENABLEGCM" Value="[CMDLINE_ENABLEGCM]" After="AppSearch" Sequence="first">CMDLINE_ENABLEGCM</SetProperty>
+
     <Feature Id='Git'>
       <Component Directory='INSTALLFOLDER'>
-        <RegistryValue Root='HKLM' Key='Software\Git' Name='InstallFolder' Value='[INSTALLFOLDER]' Type='string' />
+        <RegistryValue Root='HKLM' Key='SOFTWARE\GitForWindows' Name='InstallFolder' Value='[INSTALLFOLDER]' Type='string' />
```

Comment:
I *think* we talked about this in the past, and I seem to remember that you pointed out that our InnoSetup-based installer uses `SOFTWARE\GitForWindows`. If I remember correctly, would you kindly add a note to the commit message that this change is in line with the InnoSetup installer?